modules/capi-creating-infrastructure-resource.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="capi-creating-infrastructure-resource_{context}"]
+= Creating a Cluster API infrastructure resource
+
+You can create a provider-specific infrastructure resource by creating a YAML manifest file and applying it with the {oc-first}.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster.
+
+* You have enabled the use of the Cluster API.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have obtained the cluster ID value.
+
+* You have created and applied the cluster resource.
+
+.Procedure
+
+. Create a `<infrastructure_resource_file>.yaml` file similar to the following:
++
+--
+[source,yaml]
+----
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: <infrastructure_kind> # <1>
+metadata:
+  name: <cluster_name> # <2>
+  namespace: openshift-cluster-api
+spec: # <3>
+----
+<1> Specify the infrastructure kind for the cluster. This value must match the value for your platform. The following values are valid:
+* `AWSCluster`: The cluster is running on {aws-first}.
+* `GCPCluster`: The cluster is running on {gcp-first}.
+<2> Specify the name of the cluster.
+<3> Specify the details for your environment. These parameters are provider specific. For more information, see the sample Cluster API infrastructure resource YAML for your provider.
```

Comment:
here should have a link for `Cluster API infrastructure resource YAML for your provider`?